README.md

```diff
@@ -218,4 +258,5 @@ sessionManager.request(...)
 | interceptIOS | Boolean  |   false   |     no      | Intercept native iOS requests                                |
 | disableShake | Boolean  |   false   |     no      | Set to true to disable shake feature to display Netwatch     |
 | maxRequests  |  Number  |    100    |     no      | Maximum requests displayed                                   |
+| reduxConfig  |  Object  |    {}     |     no      | Extra infos for Redux Action                                 |
```

Comment:
default value is currently undefined

 src/index.tsx l.200

Reply:
Ok, at last, I added a default value corresponding to the readme. Also iconMapper that receive this props is defined to {} by default. (see ReduxLogger.tsx l.8) and I prefer avoid undefined value.